services/app-web/.stylelintrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I haven't been able to get this to work. I enabled vscode-stylelint to test this, are you using something different?

Reply:
I was running stylelint in app-web manually `npx styelint **/*.scss`-  let me surface that as a yarn command also and document. It should be hooked into lint-staged as well (precommit script) I'll double check that.